### ISSUE TITLE
Fan stop not working

### ISSUE THREAD

**Issue by antal-horvath** [Rem0o/FanControl.Releases#119]
I applied `Detect fans stop % automatically...` with the result for one of my fans: `never stopped`.
I know it is possible that this fan stops since FanXpert4 was able to with its feature "auto-fan stop".

My goal is to only start this fan when temps are higher than e.g. 50°C. Assigning the relative fan speed of 0% to temps lower than 50°C, however, does not work. 0% fan speed probably does not correspond to 0 power consumption.

Is this a feature that FanControl does not support yet or am I doing something wrong? I'm happy to provide more info if necessary.

**Comment by Rem0o** [comment]
Well, I can't go lower than 0, so if it doesn't stop from there, it isn't supported.

**Comment by felipebutcher** [comment]
@antal-horvath I guess this was not a very good answer you got here from the owner.

I have the same issue. Motherboard is an Asus TUF B460M-plus. My fans are lian-li and they support speed control and being stopped.

After a lot of testing. I realized the this software here is only able to stop the fans, when I have Asus Fan Expert installed and "fan auto stop" enable.

Problem is: remoo fancontrol can only stop the fans when fan expert is enable so both softwares controls the fans. if i uninstall fan expert, remoo fancontrol can't stop them anymore. so unfortunately I had to keep using fan control although i wanted to use fancontrol.
